tests/distributions/conftest.py

```diff
@@ -3,12 +3,12 @@
 import math
 
 import numpy as np
-import pytest
 import scipy.stats as sp
 
 import pyro.distributions as dist
+import pytest
```

Comment:
Run `make format` to automatically sort headers. (This header should be in the middle group because it is third-party. The groups are: __future__, builtins, third-party, local. Sorting is alphabetical within each group.)

Reply:
Hmm, yeah, I'm wondering if I'm not picking up on a config file or something? When I ran `make format` it actually changed a lot of other, unrelated files (and this change is from the output of `make format` on `conftest.py`, which I added since I was modifying the file some).

Reply:
Maybe try `pip install -U isort`?